src/rules/no-unused-modules.ts

```diff
@@ -486,7 +479,7 @@ export = createRule<Options[], MessageId>({
   defaultOptions: [],
   create(context) {
     const {
-      src,
+      src = [process.cwd()],
       ignoreExports = [],
```

Comment:
Benefits of putting the default value at this higher level:

- it flows into both `resolveFiles()` calls (lines 320 and 555) and is future-proof,
- it's next to the default value for `ignoreExports`, and
- we can drop the now-redundant `getSrc()` function.